server/src/test/java/org/elasticsearch/index/mapper/MultiFieldTests.java

```diff
@@ -227,7 +227,7 @@ public void testUnmappedLegacyFieldsUnderKnownRootField() throws Exception {
             b.startObject("name");
             b.field("type", "keyword");
             b.startObject("fields");
-            b.startObject("subfield").field("type", "text").endObject();
+            b.startObject("subfield").field("type", CompletionFieldMapper.CONTENT_TYPE).endObject();
```

Comment:
now that we support text fields, we can't use it anymore to check "placeholder" functionality. Instead we use another unsupported field.